doc/api/tty.md

```diff
@@ -2,13 +2,20 @@
 
     Stability: 2 - Stable
 
-The `tty` module houses the `tty.ReadStream` and `tty.WriteStream` classes. In
-most cases, you will not need to use this module directly.
+The `tty` module provides the `tty.ReadStream` and `tty.WriteStream` classes.
+In most cases, it will not be necessary or possible to use this module directly.
+However, it can be accessed using:
 
-When Node.js detects that it is being run inside a TTY context, then `process.stdin`
-will be a `tty.ReadStream` instance and `process.stdout` will be
-a `tty.WriteStream` instance. The preferred way to check if Node.js is being run
-in a TTY context is to check `process.stdout.isTTY`:
+```js
+const tty = require('tty');
+```
+
+When Node.js detects that it is being run inside a text terminal ("TTY")
+context, the `process.stdin` will, by default, be initialized as an instance of
+`tty.ReadStream` and both `process.stdout` and `process.stderr` will, by
+default be instances of `tty.WriteStream`. The preferred method of determining
+whether Node.js is being run within a TTY context is to check the boolean
+`process.stdout.isTTY` property:
```

Comment:
Note: this won't be boolean if stdout isn not a TTY instance, instead it will be `undefined`.